s/babylon/camera/utils/on-camera-rotate.ts

```diff
@@ -0,0 +1,12 @@
+import {v2, V2} from "../../../utils/v2.js"
+
+
+export function onCameraRotate({getUpdateStarters, addMouseforce, updateCameraRotation}: {
+		getUpdateStarters: () => {movement: V2}
+		addMouseforce: (mouseForce: V2) => void
+		updateCameraRotation: () => void
+}) {
+		const {movement: [x, y]} = getUpdateStarters()
+		addMouseforce([x, -y])
+		updateCameraRotation()
```

Comment:
this line about updateCameraRotation seems very suspicious -- it must be relying on side-effects, so it's modifying things -- we need to rewrite this to be more pure